packages/plugin-core/src/test/suite-integ/CapitalizeText.test.ts

```diff
@@ -0,0 +1,123 @@
+import { ENGINE_HOOKS } from "@dendronhq/common-test-utils";
+import _ from "lodash";
+import { ExtensionContext, Position, Selection } from "vscode";
+import { CapitalizeTextCommand } from "../../commands/CapitalizeText";
+import { VSCodeUtils } from "../../utils";
+import { expect } from "../testUtilsv2";
+import { runLegacyMultiWorkspaceTest, setupBeforeAfter } from "../testUtilsV3";
+
+suite("notes", function () {
```

Comment:
can the name here be changed to the method under test?

Reply:
Yes - I'll update it shortly.